package/MDAnalysis/coordinates/base.py

```diff
@@ -2029,6 +2038,33 @@ def _apply_transformations(self, ts):
         return ts
 
 
+class _AsciiPickle(object):
+    # mixin to allow ascii files to pickle it up with the pros
+    def __getstate__(self):
+        # Shallow copy of state of self
+        # shallow ie don't recursively copy all objects,
+        # just copy the references that __dict__ holds
+        stuff = self.__dict__.copy()
+        # don't pass the file handle over
+        del stuff['_f']
+        # instead pass enough metadata to reconstruct
+        stuff['_pickle_fn'] = self._f.name
+        # TODO: what other state does Reader hold?
+        # TODO: reconstruct file handle position
+        return stuff
+
+    def __setstate__(self, state):
+        self.__dict__.update(state)
+        self._f = anyopen(self._pickle_fn)
+
+        del self._pickle_fn
+
+class _BAsciiPickle(_AsciiPickle):
```

Comment:
This is kinda ugly, but to unpickle the file handle you need to remember the mode it was opened in. This isn't currently possible afaik